api/base/serializers.py

```diff
@@ -716,7 +731,7 @@ def to_representation(self, obj, envelope='data'):
         ])
 
         embeds = self.context.get('embed', {})
-
+        esi = self.context.get('esi', {})
```

Comment:
This makes me think that some value in `esi` will be dereferenced below, when in fact it's just a boolean. I think it'd be more readable in the future if it were `enable_esi` and the default were `False` instead of `{}`.